src/style.css

```diff
@@ -1 +1,17 @@
 /* styles go here */
+div {
```

Comment:
add classes to add styles for the elements insead of tags

Reply:
fixed the mistakes.